kwanj-k/store_manager
# #161777871 Add product to cart and get all cart items

### What does this PR do?
Consumes the add product to cart and get a cart endpoints

#### Description of Task to be Completed ?
A user should be able to add a particular product to cart and view a pending cart

### How should this be manually tested?
After cloning the repo, git pull the branch ft-cart-#161777871

* Then open the index.html file.
    ```
      Create a store and log in.
      Add a product and add it to cart.
      Click on the cart to see the pending cart.
    ```

### What are the relevant Pivotal Tracker stories?
#161777871
